application/models/Gsheet_Interface_Model.php

```diff
@@ -0,0 +1,41 @@
+<?php
+defined('BASEPATH') OR exit('No direct script access allowed');
+
+class Gsheet_Interface_Model extends CI_Model {
+
+    function getCurrentWorkingDir() {
+        // For Windows 
+        $dir = shell_exec('echo %cd%');
+        // For MacOS / Linux / CentOS server
+        // $dir = shell_exec('pwd');
+        $stripped = trim($dir);
+        return $stripped;
+    }
+
+    function google_test() {
```

Comment:
Just wondering if this is the function that we can call to get values from google sheet? So providing different id will give us values from different google sheet? I am just curious as to how this works, cause currently this function name is pretty confusing and makes me feel that this is just a testing function.

Reply:
Yes, this is indeed a testing function. This story is only for setting up the Google Sheets API. For other stories that need to use this API they will need to manipulate this function or write their own ones for their story I think.

Reply:
Ok, cool. In this case, this is all good. Thanks for replying.